server/feed/dao.js

```diff
@@ -2,19 +2,20 @@ import sequelize_pkg from 'sequelize';
 const { Op } = sequelize_pkg;
 import { Post, Comment, User } from '../db/models.js';
 
-export async function getAllPosts(searchText) {
-  if (searchText) {
-    return Post.findAll({
-      include: User,
+export async function getAllPosts(searchText, sortOrder) {
```

Comment:
Good practice in DAO is to have functions in a general form. Meaning for this case to have only 1 parameter "options" that include every possible option. In this way, each function can be used for multiple purposes and adding a new option does not require a change in all the use cases. But for this project, it doesn't matter.